appveyor.yml

```diff
@@ -11,6 +11,14 @@ init:
           Update-AppveyorBuild -Version ($env:APPVEYOR_REPO_TAG_NAME).TrimStart("v")
       }
 
+dotnet_csproj:
+  patch: true
+  file: '**\*.csproj'
+  #version: '{version}'
+  #package_version: '{version}'
+  #assembly_version: '{version}'
+  #file_version: '{version}'
+
 build_script:
   - dotnet build ./NVelocity.sln -c Release 
```

Comment:
Not sure why but the assemblies in the package are debug not release, `NVelocity.dll` has:

```
[assembly: AssemblyConfiguration("Debug")]
[assembly: Debuggable(
    DebuggableAttribute.DebuggingModes.Default |
    DebuggableAttribute.DebuggingModes.DisableOptimizations |
    DebuggableAttribute.DebuggingModes.IgnoreSymbolStoreSequencePoints |
    DebuggableAttribute.DebuggingModes.EnableEditAndContinue)]
```